docs/messaging.rst

```diff
@@ -1122,6 +1122,41 @@ Frontends can choose how they update prior outputs (or if they regard this as a
 regular ``display_data`` message). Within the jupyter and nteract_ notebooks,
 all displays that match the ``display_id`` are updated (even if there are multiple).
 
+Transient Display Data
+----------------------
+
+.. versionadded:: 5.3
+
+This message is very similar to `display_data` but its is content is meant to be
```

Comment:
I'd word this as:

> The `transient_display_data` message has the same structure as `display_data` with one addition, `title`, as well as a different intent. These messages are not to be persisted to notebooks or other formats. Examples of how this could be used is status of a long calculation as well debug information.